src/Cake.Common/IO/DirectoryAliases.cs

```diff
@@ -99,6 +129,31 @@ public static void DeleteDirectories(this ICakeContext context, IEnumerable<Dire
         [CakeAliasCategory("Delete")]
         public static void DeleteDirectories(this ICakeContext context, IEnumerable<string> directories, bool recursive = false)
```

Comment:
@mholo65 I am on the fence on this one.  They are still valid, as they can be easily mapped into the new way of doing things, so part of me says to leave them.  Having said that, would also be happy to see them removed.